### PR TITLE
Add `DenyAll` and `AllowAll` access checkers

### PR DESCRIPTION
**Pull request by vjik**
| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ✔️
| Breaks BC?    | ❌
| Fixed issues  | https://github.com/yiisoft/rbac/issues/85

### CODE REVIEW

**Review by devanych — APPROVED** [review]
Maybe add `.phpunit-watcher.yml` file?

**Comment by vjik** [comment]
> Maybe add `.styleci.yml` and `.phpunit-watcher.yml` files?

Done

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,10 +1,11 @@
 # Yii Access
 
-## 1.0.1 under development
+## 1.1.0 under development
 
-- Chg #9: Type of `$userId` in the method `AccessCheckerInterface::userHasPermission()` is specified (@roxblnfk)
+- New #15: Add `DenyAll` and `AllowAll` access checkers (vjik)
+- Chg #9: Type of `$userId` in the method `AccessCheckerInterface::userHasPermission()` is specified (roxblnfk)
```
Comment (samdark) [inline CHANGELOG.md]:
```suggestion
- Chg #9: Type of `$userId` in the method `AccessCheckerInterface::userHasPermission()` is now specified (roxblnfk)
```

File: README.md [hunk 2]
```diff
@@ -8,9 +8,15 @@
...
+[![Scrutinizer Code Quality](https://scrutinizer-ci.com/g/yiisoft/access/badges/quality-score.png?b=master)](https://scrutinizer-ci.com/g/yiisoft/access/?branch=master)
+[![Code Coverage](https://scrutinizer-ci.com/g/yiisoft/access/badges/coverage.png?b=master)](https://scrutinizer-ci.com/g/yiisoft/access/?branch=master)
+[![Mutation testing badge](https://img.shields.io/endpoint?style=flat&url=https%3A%2F%2Fbadge-api.stryker-mutator.io%2Fgithub.com%2Fyiisoft%2Faccess%2Fmaster)](https://dashboard.stryker-mutator.io/reports/github.com/yiisoft/access/master)
+[![static analysis](https://github.com/yiisoft/access/workflows/static%20analysis/badge.svg)](https://github.com/yiisoft/access/actions?query=workflow%3A%22static+analysis%22)
+[![type-coverage](https://shepherd.dev/github/yiisoft/access/coverage.svg)](https://shepherd.dev/github/yiisoft/access)
 
 This package provides an interface for checking if certain user has certain permission. Optional parameters could be passed
-for fine-grained access checks.
+for fine-grained access checks. Out of the box are available `DenyAll` and `AllowAll` implementations.
```
Comment (samdark) [inline README.md]:
```suggestion
for fine-grained access checks. Additionally, `DenyAll` and `AllowAll` implementations are available out of the box.
```

**Review by samdark — APPROVED** [review]

**Pull request merged**